tests/suites/manual/deploy_manual_aws.sh

```diff
@@ -1,22 +1,24 @@
 create_vpc() {
+	set -e
```

Comment:
This is wrong.

You should be using `set_verbosity` and not doing this directly. @jack-w-shaw can you do a follow-up PR to fix this.